core/src/codechain_machine.rs

```diff
@@ -92,15 +92,13 @@ impl CodeChainMachine {
             Action::SetShardUsers {
                 ..
             } => params.min_set_shard_users_transaction_cost(),
-            Action::Custom {
-                ..
-            } => params.min_custom_transaction_cost(),
             Action::ShardStore {
                 ..
             } => {
                 // FIXME
                 0
             }
+            _ => params.min_custom_transaction_cost(),
```

Comment:
Will you change the name `min_custom_transaction_cost` later?

Reply:
ditto.